### PR TITLE
Do not quote unicode references in CSS

### PR DESCRIPTION
**Pull request by i-like-robots**
Zero padded unicode references get proper munged by the sass loader.

screen shot 2016-04-14 at 17 07 37

screen shot 2016-04-14 at 17 15 34

### CODE REVIEW

File: client/components/related-content/main.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ module.exports = {
 
 				new VideoStream(relatedItemsEl, {
 					items: json.items.slice(0, 2),
-					itemTpl: '<li data-trackable="related item foobar">' + require('../video-stream/card.html') + '</li>',
```
Comment (matthew-andrews) [inline client/components/related-content/main.js]: foobar??

Comment (i-like-robots) [inline client/components/related-content/main.js]: foobar.

(webpack will only re-build on watch when something _actually_ changes so... whoops)

**Comment by andygnewman** [comment]
👍

**Comment by i-like-robots** [comment]
Hmm, this doesn't actually work quite right... requires further investigation.

File: client/components/related-content/main.js (hunk 1 above)
Comment (matthew-andrews) [inline client/components/related-content/main.js]: lol ok

**Pull request merged**